town.logic

```diff
@@ -1,2 +1,44 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node Main:
+      chest CafeLady
+      door Bakery
+      door East
+      door FigurineHouseEast
+      door FigurineHouseWest
+      door HearthLedge
+      door HearthNorth
+      door HearthSOuth
+      door Julietta
+      door MayorEast
+      door MayorWest
+      door North
+      door Rem
+      door SchoolEast
+      door SchoolWest
+      door Simon
+      door South
+      door Stockwell
+      door Swiftblade
+      door Well
+      door West
```

Comment:
This door requires a bidirectional split: it's possible to go from the west to the town even with the guard, but going from the town to the west requires items that let the guard let you pass (CanSpin, Cape, or Flippers, I believe). (For now use `item CanSpin`; we may revisit and make it a helper flag in the future, but I think we're heading the pseudo-item route for right now.)